.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
   - repo: https://github.com/psf/black
-    rev: 19.3b0
```

Comment:
Hi @patrikgrenfeldt, is there a way here to remove the specific version of Black to be used by pre-commit? I've tried to remove the line but it gave me error